src/components/spreadsheet/spreadsheet.ts

```diff
@@ -174,6 +176,7 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
       toggleSidePanel: this.toggleSidePanel.bind(this),
       _t: Spreadsheet._t,
       clipboard: this.env.clipboard || instantiateClipboard(),
+      menuService: new MenuService(() => this.model.trigger("update")),
```

Comment:
```suggestion
      menuService: new MenuService(() => this.render(true)),
```
Is equivalent and makes more sense. It has nothing to do with model.
Could be optimized by taking advantage of owl reactivity